README.md

```diff
@@ -42,17 +42,22 @@ async ()=> {
 
 #### Transpilation / Polyfill
 
-There is no way of transpilating the actual effect of this proposal. But instead we can write a polyfill that keeps API consistent while it's using the main thread for parsing and stringifying tasks.
+There is no way of transpilating the actual effect of this proposal. But instead we can write a polyfill that keeps API consistent while it's using the main thread for parsing and stringifying tasks. We are using promises to perform the sync operation in the next processor tick.
 
 
 ```js
 JSON.stringifyAsync = JSON.stringifyAsync || function (object) {
-  return new Promise(resolve => resolve(JSON.stringify(object)));
+  return new Promise(resolve => {
```

Comment:
^ right, due to the other args - if the async version supports them - https://github.com/mohsen1/async-json/pull/7/files#r36045340 is good.